src/app/mail/mail-detail/mail-detail.component.ts

```diff
@@ -174,6 +195,7 @@ export class MailDetailComponent implements OnInit, OnDestroy {
     if (!this.mailOptions[child.id]) {
       this.mailOptions[child.id] = {};
     }
+    this.handleEmailLinks();
```

Comment:
May be this should be after Line# 192? After the contents have been decrypted.

Reply:
yes sure.